packages/lesswrong/lib/collections/books/schema.ts

```diff
@@ -90,7 +90,20 @@ const schema: SchemaType<DbBook> = {
     foreignKey: "Sequences",
     optional: true,
   },
-
+  sequencesGridDisplay: {
```

Comment:
Nitpick: This field name is a noun phrase, but it should be a verb phrase because it's a flag.

(This is one of the unwritten conventions that fails to show up in style guides because it's hard to articulate, but which most programmers pick up linguistically.)